packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -480,6 +481,947 @@ describe('postgresql - missing database', () => {
   })
 })
 
+describe('postgresql views re-introspection warnings', () => {
+  const connectionString = process.env.TEST_POSTGRES_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
+
+  function computeSetupParams(warningCode: EngineArgs.ViewWarningCodes) {
+    const setupParams: SetupParams = {
+      connectionString,
+      // Note: at this location there is a setup.sql file
+      // which will be executed a SQL file so the database is not empty
+      dirname: path.join(
+        __dirname,
+        '..',
+        '__tests__',
+        'fixtures',
+        're-introspection',
+        'postgresql',
+        `views-warning-${warningCode}`,
+      ),
+    }
+    return setupParams
+  }
+
+  describe('postgresql views 21', () => {
+    const warningCode = 21
+    const setupParams = computeSetupParams(warningCode)
+
+    beforeAll(async () => {
+      await tearDownPostgres(setupParams!).catch((e) => {
+        console.error(e)
+      })
+    })
+
+    beforeEach(async () => {
+      await setupPostgres(setupParams!).catch((e) => {
```

Comment:
```suggestion
      await setupPostgres(setupParams).catch((e) => {
```

I don't think `setupParams` are nullable.

Reply:
As far as I can tell, they aren't, not sure what the reasoning was for this